sim/druid/feral/TestFeral.results

```diff
@@ -641,8 +641,8 @@ dps_results: {
 dps_results: {
  key: "TestFeral-AllItems-FleetShadowspiritDiamond"
  value: {
-  dps: 31809.54569
-  tps: 43875.63602
+  dps: 31851.34871
+  tps: 43729.52774
  }
 }
```

Comment:
Feral Swiftness incorrectly stacking with the meta gem is likely why there is a test result change here. I think the right way to solve this is to register an `ExclusiveEffect` category for passive movement speed buffs in `core`, and then call the same `ExclusiveEffect` wrappers that we use for non-stacking buffs/debuffs when registering the enchants, meta gem, and Feral Swiftness effect, just with different priority levels so that the strongest one is applied.